src/components/Key.vue

```diff
@@ -14,7 +14,7 @@
   @Component({
     props: {
       regions: Array as new () => string[],
-      regionColorScale: Object as () => d3.ScaleOrdinal<string, string>,
+      regionColorScale: Function as () => d3.ScaleOrdinal<string, string>,
```

Comment:
Hier krieg ich nen Error
> Conversion of type 'FunctionConstructor' to type '() => ScaleOrdinal<string, string>' may be a mistake because neither type sufficiently overlaps with the other. If this was intentional, convert the expression to 'unknown' first.